web/styles/app_components.css

```diff
@@ -552,8 +552,8 @@ input.settings_text_input {
 .invite-stream-notice {
     position: relative;
     display: block;
-    background-color: hsl(46deg 63% 95%);
-    border: 1px solid hsl(49deg 20% 84%);
+    background-color: var(--color-tip-background);
```

Comment:
Stick with the pattern of leading with the property: `--color-background-tip`; there's a few like that in this commit.

Reply:
Updated.